Source/gg2/Scripts/Sound/calculateVolume.gml

```diff
@@ -3,13 +3,22 @@
     xmid = view_xview[0] + view_wview[0]/2;
     ymid = view_yview[0] + view_hview[0]/2;
     
-    dist = sqrt((xmid-argument0)*(xmid-argument0) + (ymid-argument1)*(ymid-argument1));
+    dist = sqrt(sqr(xmid-argument0) + sqr(ymid-argument1));
     
+    /*
     if(dist<300) {
         return 1;
     } else if(dist>1500) {
         return 0;
     } else {
         return ((1500-dist)/1200);
     }
-}
\ No newline at end of file
+    */
+    global.dist = dist;
+    if (dist > 10) {
+        return (1500/dist);
+    }
+    else {
+        return 1;
+    }
+}

```

Comment:
Why is this changed?

Reply:
Oh I remember now, this was an attempt at 1. Figuring put how the panning code worked, 2. Fix audio being based on the center of the screen, which causes issues when aiming as sniper
This should be removed, my bad